deps/Makefile

```diff
@@ -79,7 +79,7 @@ endif
 
 # If the top-level Makefile is called with environment variables,
 # they will override the values passed above to ./configure
-MAKE_COMMON := DESTDIR="" prefix=$(build_prefix) bindir=$(build_bindir) libdir=$(build_libdir) libexecdir=$(build_libexecdir) datarootdir=$(build_datarootdir) includedir=$(build_includedir) sysconfdir=$(build_sysconfdir) O=
+MAKE_COMMON := DESTDIR="" prefix=$(build_prefix) libdir=$(build_libdir) libexecdir=$(build_libexecdir) datarootdir=$(build_datarootdir) includedir=$(build_includedir) sysconfdir=$(build_sysconfdir) O=
```

Comment:
that's a little broken since some of the makefile-only deps are using bindir for dlls. will check again if it's only the ones easily under our control. see commit message

> Do not override bindir in MAKE_COMMON since only autotools-using deps
install any executables, and some of the makefile-only deps (openlibm,
openspecfun) install dlls into bindir on windows so we don't want to
set that to usr/tools